src/calculateRentalCost.js

```diff
@@ -4,7 +4,21 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const longTermRent = 7;
```

Comment:
```suggestion
  const LONG_TERM_RENT = 7;
```
https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md